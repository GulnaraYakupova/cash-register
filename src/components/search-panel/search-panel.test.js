import React from 'react';
import SearchPanel from './search-panel';
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

Enzyme.configure({ adapter: new Adapter() });

it ('Should render SearchPanel component', () => {
  const component = shallow(
    <SearchPanel 
      onSearchPanelType={()=>{}}
    />
  );
  expect(component).toMatchSnapshot();
});