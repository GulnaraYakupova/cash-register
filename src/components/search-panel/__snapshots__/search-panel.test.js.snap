// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render SearchPanel component 1`] = `ShallowWrapper {}`;
